doc/classes/MovieWriter.xml

```diff
@@ -6,7 +6,7 @@
 	<description>
 		Godot can record videos with non-real-time simulation. Like the [code]--fixed-fps[/code] [url=$DOCS_URL/tutorials/editor/command_line_tutorial.html]command line argument[/url], this forces the reported [code]delta[/code] in [method Node._process] functions to be identical across frames, regardless of how long it actually took to render the frame. This can be used to record high-quality videos with perfect frame pacing regardless of your hardware's capabilities.
 		Godot has 2 built-in [MovieWriter]s:
-		- AVI container with MJPEG for video and uncompressed audio ([code].avi[/code] file extension). Lossy compression, medium file sizes, fast encoding. The lossy compression quality can be adjusted by changing [member ProjectSettings.editor/movie_writer/mjpeg_quality]. The resulting file can be viewed in most video players, but it must be converted to another format for viewing on the web or by Godot with [VideoStreamPlayer]. MJPEG does not support transparency. AVI output is currently limited to a file of 4 GB in size at most.
+		- AVI container with MJPEG for video and uncompressed audio ([code].avi[/code] file extension). This requires the [code]jpg[/code] module to be enabled at compile time (default behavior). Lossy compression, medium file sizes, fast encoding. The lossy compression quality can be adjusted by changing [member ProjectSettings.editor/movie_writer/mjpeg_quality]. The resulting file can be viewed in most video players, but it must be converted to another format for viewing on the web or by Godot with [VideoStreamPlayer]. MJPEG does not support transparency. AVI output is currently limited to a file of 4 GB in size at most.
```

Comment:
What a mouthful. I understand why this remark is here, but it ruins the sentence's flow, because you'd expect the description to go into detail right away. Plus it's the default behaviour, so most don't need to read this. It could be moved at the bottom of the description in its own note.

Reply:
Fair enough, moved to a note.